OpencartTest/Mykhailo.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="wishList">
+
+
+    <test name="addToWishList">
+
+        <parameter name="myAccountDropdownText" value="Login"/>
+
+        <parameter name="fromSearchPageProductName" value="iPod Nano"/>
+        <parameter name="fromHomePageProductName" value="iPhone"/>
+        <parameter name="fromProductPageProductName" value="Samsung Galaxy Tab 10.1"/>
+
+        <classes>
+            <class name="test.TestWishList">
+                <methods>
```

Comment:
keep your xmls clean and don't comment out stuff
general suggestion - include whole classes, not methods 
you'll lose something important that way